WordPress/Classes/Extensions/AbstractPost+PostInformation.swift

```diff
@@ -0,0 +1,10 @@
+
+extension AbstractPost: PostInformation {
+    var isPostPrivate: Bool {
```

Comment:
Since we already have `isPrivate`, this ought to be `isPrivateOnWPCom` or such to be more specific about it does.